backend/controller/observability/deployment.go

```diff
@@ -19,6 +19,7 @@ type DeploymentMetrics struct {
 	reconciliationFailures metric.Int64Counter
 	reconciliationsActive  metric.Int64UpDownCounter
 	replicasAdded          metric.Int64UpDownCounter
+	replicasRemoved        metric.Int64UpDownCounter
```

Comment:
Should we make these regular counters again?

Reply:
yep